src/components/NavigationBar/Footer.jsx

```diff
@@ -0,0 +1,5 @@
+import NavBar from '@/components/NavigationBar/NavBar';
```

Comment:
It is a bit confuising that footer and header are in the NavigatinBar folder.

Reply:
fixed